_posts/2022-01-10-bevy-labels.md

```diff
@@ -0,0 +1,214 @@
+---
+title: How Bevy uses Rust traits for labelling
+categories:
+- rust
+- bevy
+---
+Out of curiosity I've recently started following the development of [Bevy],
+a game engine written in [Rust] that.
```

Comment:
Sentence fragment

Reply:
Ah yes, thanks! I moved the ECS description further down